app/build.gradle

```diff
@@ -425,7 +437,7 @@ android {
         // MetaStore and AppLab only apply to oculusvr builds.
         if ((store == 'metaStore' || store == "appLab") && !platform.startsWith('oculusvr'))
             variant.setIgnore(true);
-        
+
```

Comment:
nice but not really needed, let's not do it here

Reply:
Done.